README.md

```diff
@@ -1,31 +1,100 @@
 # Hiero technical governance
 
-This repository contains the `config.yaml` file that is used to configure all GitHub repositories of the organisation. https://clowarden.io is used as tool to manage the GitHub ressources and the changelist for all Hiero repos can be found [here](https://clowarden.io/audit/?organization=LFDT-Hiero).
+This repository contains the `config.yaml` file that is used to configure all GitHub repositories of the organisation.
+https://clowarden.io is used as tool to manage the GitHub resources and the change list for all Hiero repos can be found [here](https://clowarden.io/audit/?organization=LFDT-Hiero).
+
+## Definition of general roles
+
+In general the Hiero project (and the LFDT) defines 3 different types of **roles**: **Contributors**, **Committers**, and **Maintainers**.
+Independent of the role each person who contributes to Hiero must follow our **Code of Conduct** (https://github.com/hiero-ledger/.github/blob/main/CODE_OF_CONDUCT.md).
+Those roles are not assigned globally for the Hiero project but per **project**.
+In most cases a project is a single GitHub repository, but it can also be a group of repositories that are related to each other.
+A person can have multiple roles in multiple projects.
+
+### Contributors
+In general any person that does any contribution to the project is a Contributor.
+A contribution can be anything from opening an issue, to writing code, to writing documentation, etc.
```

Comment:
I recommend the following:

A contribution can be anything from opening an issue, to writing code, to writing documentation, etc:
- If a PR has gone 2 days without approvals or an Issue or Discussion has no activity for 2 days, the owner must reach our to our hiero-general Discord channel.
- If a PR or discussion has gone stale for 2 weeks, the repo maintainers can close them due to inactivity.